src/reducers/query/querySlice.spec.js

```diff
@@ -43,85 +41,84 @@ describe('reducer:query', () => {
         searchAfter: '',
         size: 25,
         sort: 'created_date_desc',
-        totalPages: 0,
         trendsDateWarningEnabled: false,
       });
     });
   });
 
-  describe('COMPLAINTS_RECEIVED actions', () => {
-    it('updates total number of pages', () => {
-      const payload = {
-        data: {
-          _meta: {
-            break_points: {
-              2: [2, 2],
-              3: [3, 2],
-              4: [4, 2],
-              5: [5, 2],
-            },
-          },
-          hits: {
-            total: { value: 10000 },
-          },
-        },
-      };
-
-      state = {
-        ...queryState,
-        page: 10,
-        size: 100,
-        totalPages: 5,
-      };
-
-      expect(target(state, complaintsReceived(payload))).toEqual({
-        ...state,
-        breakPoints: {
-          2: [2, 2],
-          3: [3, 2],
-          4: [4, 2],
-          5: [5, 2],
-        },
-      });
-    });
-
-    it('limits the current page correctly', () => {
-      const payload = {
-        data: {
-          _meta: {
-            break_points: {
-              2: [2, 2],
-              3: [3, 2],
-              4: [4, 2],
-              5: [5, 2],
-            },
-          },
-          hits: {
-            total: { value: 10000 },
-          },
-        },
-      };
-
-      state = {
-        ...queryState,
-        page: 101,
-        size: 100,
-      };
-
-      expect(target(state, complaintsReceived(payload))).toEqual({
-        ...state,
-        breakPoints: {
-          2: [2, 2],
-          3: [3, 2],
-          4: [4, 2],
-          5: [5, 2],
-        },
-        page: 100,
-        size: 100,
-        totalPages: 5,
-      });
-    });
-  });
+  // describe('COMPLAINTS_RECEIVED actions', () => {
```

Comment:
Can this commented code be deleted?

Reply:
That reminds me, I commented out these tests since these need to be moved over to validate the API.